components/typography/stories/typography.stories.js

```diff
@@ -18,7 +18,7 @@ import {
 import { TypographyGroup } from "./typography.test.js";
 
 /**
- * Spectrum typography is broken out into several separate components: heading, body, detail, and code. Internationalized typography examples are also shown.
+ * Spectrum typography is broken out into several separate components: [heading](#heading), [body](#body), [detail](#detail), and [code](#code). [Internationalized typography examples](#internationalized%20body) are also shown.
```

Comment:
I couldn't get the internationalized link to work for a while. This is the href I was able to copy from the heading itself, but this feels weird with the `%20`. 

This link doesn't work on production currently, but if we kept the MDX file we could update it really easily. It is linked to a heading `internationalization` that doesn't exist:
<img width="321" alt="Screenshot 2025-03-27 at 12 49 44 PM" src="https://github.com/user-attachments/assets/7c1ea558-1b36-4e5b-ba8b-c3f7f4970225" />

But it should link to the `internationalized-typography` heading and be good to go.
<img width="326" alt="Screenshot 2025-03-27 at 12 49 31 PM" src="https://github.com/user-attachments/assets/48a6e0b1-332f-4aad-bdd7-979a7093f187" />